doc/security/specs/identities_and_root_keys/_index.md

```diff
@@ -13,8 +13,14 @@ This document also defines a non-cryptographic **Device Identifier** to
 facilitate silicon tracking during manufacturing flows. The Device Identifier is
 also mixed into the Creator Identity.
 
+Further, this scheme is compatible with the [Open DICE profile](https://www.google.com/url?q=https://pigweed.googlesource.com/open-dice/%2B/HEAD/docs/specification.md%23Open-Profile-for-DICE&sa=D&source=editors&ust=1625179205465000&usg=AOvVaw1qqNWzsTpWSkXPnZGxBEkM).
```

Comment:
nit: Update link to 

https://pigweed.googlesource.com/open-dice/+/refs/heads/main/docs/specification.md

Reply:
ah yes thanks.